tests/linearizability/identity/lease_ids.go

```diff
@@ -15,39 +14,151 @@
 package identity
 
 import (
+	"math/rand"
 	"sync"
 )
 
 type LeaseIdStorage interface {
-	LeaseId(int) int64
-	AddLeaseId(int, int64)
-	RemoveLeaseId(int)
+	// Add adds lease to storage.
+	Add(int64)
+
+	// Remove permanently removes lease from storage.
+	Remove(int64)
+
+	// Take returns an unused lease from storage
+	Take() int64
+
+	// Return returns a used lease to storage so it can be used later.
+	Return(int64)
+
+	// GetRandom returns a random lease (could be in use or not)
+	GetRandom() int64
```

Comment:
GetRandom only complicates the interface without real benefit. It just a combination of `Remove(Take())`